packages/esm-billing-app/src/billable-services/create-edit/add-billable-service.component.tsx

```diff
@@ -47,22 +47,23 @@ const paymentFormSchema = z.object({
   concept: z.string({ required_error: 'Concept search is required.' }),
 });
 
-type FormData = z.infer<typeof paymentFormSchema>;
-const DEFAULT_PAYMENT_OPTION = { paymentMode: '', price: '1' };
-
-const AddBillableService: React.FC = () => {
+export type FormData = z.infer<typeof paymentFormSchema>;
+const DEFAULT_PAYMENT_OPTION = { paymentMode: '', price: '1', uuid: null };
```

Comment:
```suggestion
const DEFAULT_PAYMENT_OPTION = { paymentMode: '', price: '', uuid: null };
```

why have the default to 1 for price